test/Vagrantfile

```diff
@@ -32,6 +32,7 @@ Vagrant.configure("2") do |config|
         server.vm.provision "file", source: "provision", destination: "/tmp/provision"
         server.vm.provision "shell" do |sh|
             sh.path = "./provision/runtime_install.sh"
+            sh.env = {"CILIUM_USE_ENVOY" => "#{$CILIUM_USE_ENVOY}"}
```

Comment:
We don't need the $CILIUM_USE_ENVOY variable in the Vagrantfile, so maybe do it like this:
```sh.env = {"CILIUM_USE_ENVOY" => "1"}```

Reply:
Can we just get rid of `CILIUM_USE_ENVOY`?